<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1412</width>
    <height>919</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton
{
	border: 2px solid #587394 ; 
	border-radius: 10px ; 
	padding: 10px;
	color: #2b2b2b;
	background-color: #dee3ff;
	font-weight: 600;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>730</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	width: 100px;
    height: 200px;
	background-color: #ff9ca1;
	font: 900 17px ;
}</string>
    </property>
    <property name="text">
     <string>click to close
 the window</string>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>1261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>1241</width>
      <height>631</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_2">
           <property name="text">
            <string>CheckBox</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,0,0">
         <property name="spacing">
          <number>30</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QComboBox" name="comboBox">
               <property name="maximumSize">
                <size>
                 <width>370</width>
                 <height>70</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>college of medicen</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>The choosen item is : </string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QListWidget" name="listWidget">
               <item>
                <property name="text">
                 <string>apple </string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>milon</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>banana </string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>etc...</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QPushButton" name="pushButton_3">
                 <property name="text">
                  <string>color </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="boldListWidget">
                 <property name="text">
                  <string>bold &amp; italic</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="background">
                 <property name="text">
                  <string>backgound color </string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPushButton" name="infoBox">
             <property name="text">
              <string>Info Box </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="warningButton">
             <property name="text">
              <string>Warning Box</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="aboutBox">
             <property name="text">
              <string>about box </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="QuestionBox">
             <property name="text">
              <string>Question Box</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_5">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>CLICK TO MOVE AND FILL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="styleSheet">
          <string notr="true">QLineEdit
{
	font: 200 11pt;
	padding: 10px;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1412</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWhat_d_u_want">
    <property name="title">
     <string>File </string>
    </property>
    <addaction name="action1"/>
    <addaction name="separator"/>
    <addaction name="action2"/>
    <addaction name="separator"/>
    <addaction name="action3"/>
   </widget>
   <widget class="QMenu" name="menutst_2">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="action1_2"/>
    <addaction name="separator"/>
    <addaction name="action2_2"/>
    <addaction name="separator"/>
    <addaction name="action3_2"/>
    <addaction name="separator"/>
    <addaction name="action4_2"/>
    <addaction name="actionprint"/>
   </widget>
   <widget class="QMenu" name="menuabout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionabout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuhelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionhelp"/>
   </widget>
   <widget class="QMenu" name="menufont">
    <property name="title">
     <string>font</string>
    </property>
    <addaction name="actionfont"/>
    <addaction name="actioncolor"/>
   </widget>
   <addaction name="menuWhat_d_u_want"/>
   <addaction name="menutst_2"/>
   <addaction name="menuabout"/>
   <addaction name="menuhelp"/>
   <addaction name="menufont"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action1"/>
   <addaction name="action2"/>
   <addaction name="action3"/>
   <addaction name="action1_2"/>
   <addaction name="action2_2"/>
   <addaction name="action3_2"/>
   <addaction name="action4_2"/>
   <addaction name="actionprint"/>
   <addaction name="actionhelp"/>
   <addaction name="actionabout_Qt"/>
   <addaction name="actionfont"/>
   <addaction name="actioncolor"/>
  </widget>
  <action name="action1">
   <property name="icon">
    <iconset resource="imagesForIcons.qrc">
     <normaloff>:/imageFI/folder-open-regular.svg</normaloff>:/imageFI/folder-open-regular.svg</iconset>
   </property>
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="action2">
   <property name="icon">
    <iconset resource="imagesForIcons.qrc">
     <normaloff>:/imageFI/floppy-disk-solid.svg</normaloff>:/imageFI/floppy-disk-solid.svg</iconset>
   </property>
   <property name="text">
    <string>save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action3">
   <property name="icon">
    <iconset resource="imagesForIcons.qrc">
     <normaloff>:/imageFI/xmark-solid.svg</normaloff>:/imageFI/xmark-solid.svg</iconset>
   </property>
   <property name="text">
    <string>quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action1_2">
   <property name="icon">
    <iconset resource="imagesForIcons.qrc">
     <normaloff>:/imageFI/copy-solid.svg</normaloff>:/imageFI/copy-solid.svg</iconset>
   </property>
   <property name="text">
    <string>copy</string>
   </property>
  </action>
  <action name="action2_2">
   <property name="icon">
    <iconset resource="imagesForIcons.qrc">
     <normaloff>:/imageFI/paste-regular.svg</normaloff>:/imageFI/paste-regular.svg</iconset>
   </property>
   <property name="text">
    <string>paste</string>
   </property>
  </action>
  <action name="action3_2">
   <property name="icon">
    <iconset resource="imagesForIcons.qrc">
     <normaloff>:/imageFI/pen-solid.svg</normaloff>:/imageFI/pen-solid.svg</iconset>
   </property>
   <property name="text">
    <string>update </string>
   </property>
  </action>
  <action name="action4_2">
   <property name="icon">
    <iconset resource="imagesForIcons.qrc">
     <normaloff>:/imageFI/trash-solid.svg</normaloff>:/imageFI/trash-solid.svg</iconset>
   </property>
   <property name="text">
    <string>delete</string>
   </property>
  </action>
  <action name="actionhelp">
   <property name="icon">
    <iconset resource="imagesForIcons.qrc">
     <normaloff>:/imageFI/circle-info-solid.svg</normaloff>:/imageFI/circle-info-solid.svg</iconset>
   </property>
   <property name="text">
    <string>help </string>
   </property>
  </action>
  <action name="actionabout_Qt">
   <property name="icon">
    <iconset resource="imagesForIcons.qrc">
     <normaloff>:/imageFI/question-solid.svg</normaloff>:/imageFI/question-solid.svg</iconset>
   </property>
   <property name="text">
    <string>about Qt</string>
   </property>
  </action>
  <action name="actionprint">
   <property name="icon">
    <iconset resource="imagesForIcons.qrc">
     <normaloff>:/imageFI/print-solid.svg</normaloff>:/imageFI/print-solid.svg</iconset>
   </property>
   <property name="text">
    <string>print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionfont">
   <property name="icon">
    <iconset resource="imagesForIcons.qrc">
     <normaloff>:/imageFI/font-solid.svg</normaloff>:/imageFI/font-solid.svg</iconset>
   </property>
   <property name="text">
    <string>font</string>
   </property>
  </action>
  <action name="actioncolor">
   <property name="icon">
    <iconset resource="imagesForIcons.qrc">
     <normaloff>:/imageFI/palette-solid.svg</normaloff>:/imageFI/palette-solid.svg</iconset>
   </property>
   <property name="text">
    <string>color</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="imagesForIcons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>562</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
